chain/network/src/network_protocol/network.proto

```diff
@@ -343,4 +343,6 @@ message PeerMessage {
     EpochSyncFinalizationRequest epoch_sync_finalization_request = 22;
     EpochSyncFinalizationResponse epoch_sync_finalization_response = 23;
   }
+
+  google.protobuf.Timestamp tx_timestamp = 25;
```

Comment:
"_timestamp" suffix is redundant with the field type. How about sth like "sent_at"?.